### PR TITLE
Chylli/fix pod

### PR DESCRIPTION
**Pull request by chylli**

### CODE REVIEW

File: lib/Experian/IDAuth.pm [hunk 1]
```diff
@@ -2,7 +2,7 @@ package Experian::IDAuth;
 use strict;
 use warnings;
 
-our $VERSION = '2.4';
```
Comment (raunakkathuria) [inline lib/Experian/IDAuth.pm]: actually I have uploaded 2.5 (as dzil automatically increased it to 2.5) already to cpan, its waiting for upload.

**Comment by raunakkathuria** [comment]
i think we also need to update `dist.ini` by moving this https://github.com/binary-com/perl-experian-idauth/blob/master/dist.ini#L32 before https://github.com/binary-com/perl-experian-idauth/blob/master/dist.ini#L30 as per the email we received

File: lib/Experian/IDAuth.pm (hunk 1 above)
Comment (chylli) [inline lib/Experian/IDAuth.pm]: @raunakkathuria  then can I use `2.51` as the next version ? I think `2.x` will be `3.x` soon, too quickly.

**Comment by chylli** [comment]
Yes you are right. I should change  `dist.ini` in every repo :(

**Pull request closed without merging**